asciidoc/modules/con_mg_migrating-applications-to-new-redis-client.adoc

```diff
@@ -0,0 +1,187 @@
+[id="migrating-applications-to-new-redis-client_{context}"]
+= Migrating existing Redis client applications to new client
+
```

Comment:
@vietj @pmlopes I have restructured the Redis information. Please let me know if the overall structure and content reads ok :).
The top chapter is:
asciidoc/chapters/assembly_mg_changes-in-vertx-redis-client.adoc

Reply:
@pmlopes can you review the the overall chapter ?

Reply:
@pmlopes Can I resolve this?